src/Zaaksysteem/GravityForms/GravityFormsFieldSettings.php

```diff
@@ -33,9 +67,22 @@ public function addSelect($position, $formId): void
             return;
         }
 
-        // Render the supplier based options.
-        $mappingOptions = sprintf('partials/gf-field-options-%s.php', $supplier);
+        // Get the selected zaaktype from the form settings.
+        $zaaktype = $form[sprintf('%s-form-setting-%s-identifier', OWC_GZ_PLUGIN_SLUG, $supplier)];
```

Comment:
Voor de zekerheid misschien nog checken of deze variabele de juist waarde bevat?

Reply:
Ik heb hem iets getweakt maar blijf het een lelijke getter vinden @sanderdekroon kan die `gf.setting` hiervoor gebruikt worden?